migrations/sqlite/20250619092927_remove_account_name_from_passwrod_table.sql

```diff
@@ -0,0 +1,8 @@
+/* ------------------------------ */
+
+DROP INDEX IF EXISTS idx_accounts_passwords_account_name;
+
+ALTER TABLE accounts_passwords
+    DROP COLUMN account_name;
```

Comment:
Interesting
Q: in the case of SQLite there is no automatic deletion of indexes?

Reply:
Wow, live & learn